### PR TITLE
go.mod: Bump up to go 1.19 and bump up containerd to v1.7.0-beta.2

### PR DESCRIPTION
**Pull request by ktock**
Fixes: #1014

### CODE REVIEW

**Comment by ktock** [comment]
in k3s build:

```
 github.com/k3s-io/k3s/pkg/daemons/executor imports
	go.etcd.io/etcd/server/v3/embed imports
	go.opentelemetry.io/otel/semconv: module go.opentelemetry.io/otel@latest found (v1.11.2), but does not contain package go.opentelemetry.io/otel/semconv
github.com/k3s-io/k3s/pkg/daemons/executor imports
	go.etcd.io/etcd/server/v3/embed imports
	go.opentelemetry.io/otel/exporters/otlp imports
	go.opentelemetry.io/otel/sdk/metric/controller/basic imports
	go.opentelemetry.io/otel/metric/registry: module go.opentelemetry.io/otel/metric@latest found (v0.34.0), but does not contain package go.opentelemetry.io/otel/metric/registry
```

File: ipfs/go.mod [hunk 1]
```diff
@@ -1,15 +1,95 @@
...
 	github.com/ipfs/go-cid v0.1.0
 	github.com/ipfs/go-ipfs-files v0.2.0
 	github.com/ipfs/interface-go-ipfs-core v0.7.0
+	github.com/opencontainers/image-spec v1.0.3-0.20211202183452-c5a74bcca799
+)
+
+require (
+	github.com/Microsoft/go-winio v0.5.2 // indirect
+	github.com/Microsoft/hcsshim v0.9.5 // indirect
+	github.com/btcsuite/btcd v0.21.0-beta // indirect
```
Comment (AkihiroSuda) [inline ipfs/go.mod:16]: Needs explanation 
- https://github.com/containerd/nerdctl/pull/1573

Comment (AkihiroSuda) [reply ipfs/go.mod:16]: See also
- https://github.com/containerd/nerdctl/pull/1626

**Comment by AkihiroSuda** [comment]
Looks good but the commit title seems inaccurate

**Review by AkihiroSuda — APPROVED** [review]

**Pull request merged**